python/py-gdal/Portfile

```diff
@@ -20,16 +20,19 @@ long_description    This Python package and extensions are a number of tools for
 
 homepage            https://www.gdal.org
 
-checksums           rmd160  d459e0f7f60516f263f67baad215b76be22f1b1e \
-                    sha256  ad8addd58ba9c62aefc7f65d345c168736798f137e5c8f247af76fcf4862d371 \
-                    size    802628
+checksums           rmd160  92bf1a95fb5da655fe0063c7f1d49f573e63e329 \
+                    sha256  7c2a857cf903b9921357e0dfae593f2fe6f7dcd2ea42600577d5fc3c864fb066 \
+                    size    839919
 
 python.versions     38 39 310 311 312
 
 if {${name} ne ${subport}} {
     compiler.cxx_standard   2011
     compiler.thread_local_storage yes
 
+    # Setting build.cmd manually may be removed when py-numpy is >= 2
+    build.cmd "${python.bin} -m build --no-isolation --skip-dependency-check"
```

Comment:
Why? Does the package require a (pre-release) version of NumPy 2? If so, does it still work correctly like this? If so, can you just patch the pyproject.toml file?

Reply:
I agree this is a pretty peculiar solution (see background at https://github.com/OSGeo/gdal/issues/9751) . `py-gdal` "requires numpy (1.0.0 or greater)", but the "pyproject.toml file of GDAL 3.9 requires numpy >= 2.0.0rc1 (for Python >= 3.9) at build time to be able to build bindings that are compatible of both NumPy 1 and NumPy 2" according to [docs](https://pypi.org/project/GDAL). As Numpy is the only required dependency and is declared in Portfile, I'd prefer this solution (passing `--skip-dependency-check`).